spyder_notebook/widgets/dom.py

```diff
@@ -55,3 +55,11 @@ def set_input_value(self, selector, value):
         script = 'document.querySelector("%s").setAttribute("value", "%s")'
         script = script % (selector, value)
         self.evaluate(script)
+
+    def set_class_value(self, selector, classname):
+        """Sets the class of element matched by the given selector."""
+        return self.evaluate("""
+            (function () {
+            var element = document.querySelector(%s);
+            element.className = %s;
+            })();""" % (repr(selector), repr(classname)))
```

Comment:
About `repr` we can remove it, however we would need to pass the parameters with `'`, so for example instead of `"#header-container"`, we would need to pass `"'#header-container'"`. Since the click method works with `repr` I thought that use it here too was ok. About the format :+1:

Also taking this into account it would be better if the `click` method also worked with format and without `repr`?